src/Akeneo/Connectivity/Connection/back/Infrastructure/Persistence/Dbal/Repository/DbalEventsApiRequestCountRepository.php

```diff
@@ -0,0 +1,46 @@
+<?php
+declare(strict_types=1);
+
+namespace Akeneo\Connectivity\Connection\Infrastructure\Persistence\Dbal\Repository;
+
+use Akeneo\Connectivity\Connection\Domain\Webhook\Persistence\Repository\EventsApiRequestCountRepository;
+use Doctrine\DBAL\Connection as DbalConnection;
+use Doctrine\DBAL\Types\Types;
+
+/**
+ * @author    Pierre Jolly <pierre.jolly@akeneo.com>
+ * @copyright 2020 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class DbalEventsApiRequestCountRepository implements EventsApiRequestCountRepository
+{
+    private DbalConnection $dbalConnection;
+
+    public function __construct(DbalConnection $dbalConnection)
+    {
+        $this->dbalConnection = $dbalConnection;
+    }
+
+    public function upsert(\DateTimeImmutable $dateTime, int $eventCount): int
```

Comment:
```suggestion
    /**
     * Update the number of Events API requests for the current hour & minute.
     * `$dateTime` minute is used to determine the minute to update and to set the last `updated` date.
     * If the previous `updated` date is not from the current hour, then the count is reinitialized with the new count.
     */
    public function upsert(\DateTimeImmutable $dateTime, int $eventCount): int
```
Suggesting a little description to explain the query.